templates/hound.yml

```diff
@@ -1,17 +1,17 @@
 # See https://houndci.com/configuration for help.
 coffeescript:
-  # config_file: .coffeescript-style.json
+  config_file: .coffeescript-style.json
   enabled: true
 haml:
   # config_file: .haml-style.yml
   enabled: true
 javascript:
-  # config_file: .javascript-style.json
+  config_file: .javascript-style.json
   enabled: true
   # ignore_file: .javascript_ignore
 ruby:
-  # config_file: .ruby-style.yml
+  config_file: .ruby-style.yml
```

Comment:
Each of the `config_file` values in this file (or at least the ones we are actively pulling from thoughtbot/guides) should match the name we are passing from the download in `lib/suspenders/app_builder.rb`.

For example, this one should be `.rubocop.yml`…

_Or_, we change the download action to name this to `.ruby-style.yml`